src/wallet/spend.h

```diff
@@ -187,11 +187,20 @@ struct CreatedTransactionResult
  */
 util::Result<CreatedTransactionResult> CreateTransaction(CWallet& wallet, const std::vector<CRecipient>& vecSend, int change_pos, const CCoinControl& coin_control, bool sign = true);
 
+struct FundedTransactionResult
+{
+    CAmount fee;
+    int change_pos;
+
+    FundedTransactionResult(CAmount _fee, int _change_pos)
+        : fee(_fee), change_pos(_change_pos) {}
+};
```

Comment:
In 038ed838:

Instead of creating a new struct for this that duplicates part of what we have in the `CreatedTransactionResult` struct, I would make a base class for `CreatedTransactionResult` that only contains the fee, fee calc and change pos fields. Then make `CreatedTransactionResult` a child of it. And use the new base class directly here.

Reply:
Thanks, done.